Adds redirect for buienradar camera component

ties opened the pull request:
**Description:**
When camera component is mis-configured, an error includes a URL
to documentation.

However, this page did not exist due to a missing redirect.

```
2019-10-01 11:42:01 ERROR (MainThread) [homeassistant.config] Invalid config for [camera.buienradar]: [interval] is an invalid option for [camera.buienradar]. Check: camera.buienradar->interval. (See ~/.homeassistant/configuration.yaml, line 30). Please check the docs at https://home-assistant.io/components/camera.buienradar/
```

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

frenck commented:
I'll assign myself, I'm currently working on redirects throughout the whole site. Merging this now, will likely cause merge conflicts on larger PR's right now. So holding it off for a bit.

ties commented:
Ack.

I agree that "fix" it is a bit icky. The underlying issue is that the documentation URL is templated in the error log while it may be one composite page. If a change in the component is needed to have matching metadata I'll do that.

frenck commented:
@ties,

So the redirect have moved and are now in the file `_source/_redirects`, could you please adjust/move that?

ties committed:
Adds redirect for buienradar camera component

When camera component is mis-configured, an error includes a URL
to documentation.

However, this page did not exist due to a missing redirect.

frenck reviewed (approved):
Thanks, @ties 👍 

Happy Hacktoberfest 🎉
